pkg/node/nodes_test.go

```diff
@@ -0,0 +1,84 @@
+package node
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+var _ = Describe("MaxUnavailable nodes", func() {
+	type maxUnavailableCase struct {
+		nmstateEnabledNodes          int
+		maxUnavailable               intstr.IntOrString
+		expectedScaledMaxUnabailable int
+		expectedError                bool
```

Comment:
We can put here a gomega matcher
```suggestion
		expectedError                types.GomegaMatcher
```

Reply:
Good point, thanks. Done